src/partial-result-stream.js

```diff
@@ -64,7 +64,7 @@ function partialResultStream(requestFn, options) {
   var batchAndSplitOnTokenStream = checkpointStream.obj({
     maxQueued: 10,
     isCheckpointFn: function(row) {
-      return is.defined(row.resumeToken);
+      return Buffer.isBuffer(row.resumeToken);
```

Comment:
This may or may not have been a problem. It looked like we were sometimes getting empty arrays for "row.resumeToken". This changes to confirm it's an actual resume token.